### DISCUSSION TITLE
csfixer v3 support ?

### DISCUSSION THREAD

**Discussion by christophe-fms** [junichi11/netbeans-php-cs-fixer-plugin#38] [Q&A] [answered]
Is there an update programmed to support csfixer v3 ?

**Comment by junichi11** [comment, marked as answer]
Can't you run v3?
What are there missing?

**Reply by christophe-fms** [reply]
In Netbeans, i can chosse only v1 or v2 :
![image](https://user-images.githubusercontent.com/47887993/177545136-2bb0bfef-1499-4a47-aaf5-da1ad00ba290.png)
Maybe i do someting wrong...

**Reply by junichi11** [reply]
> In Netbeans, i can chosse only v1 or v2

Yes, I don't change it yet.

Did you set the path of php-cs-fixer to the Options(Tools > Options > PHP > Frameworks & Tools > PHP CS Fixer)?

**Reply by junichi11** [reply]
Add v3: https://github.com/junichi11/netbeans-php-cs-fixer-plugin/releases/tag/v0.12.0

**Reply by christophe-fms** [reply]
Thanks a lot, works fine now.Just want to send you a million thanks :-) You did the job very quickly.
